Date: Jun 27 16:45:07 2017
 
From: 0a742ec11db00fe2263a30da9cc05bff73e16691 Jun 29 20:04:10 2017

Subject: [PATCH] RDK-19472 : Key Handling for Browser Apps

	Maximum keycode support is increased from 255 to 512 for handling color keys in remote.
	
Source: COMCAST 

Upstream-Status: Pending

Signed-off-by: aansari <AzamAyub_Ansari@cable.comcast.com>
---


diff -aruN xkeyboard-config-2.11-orig/keycodes/evdev xkeyboard-config-2.11/keycodes/evdev
--- xkeyboard-config-2.11-orig/keycodes/evdev	2017-06-27 15:41:14.744375351 -0400
+++ xkeyboard-config-2.11/keycodes/evdev	2017-06-27 15:42:06.637213725 -0400
@@ -2,7 +2,7 @@
 
 default xkb_keycodes "evdev" {
 	minimum = 8;
-	maximum = 255;
+	maximum = 512;
 
         # Added for pc105 compatibility
         <LSGT> = 94;
@@ -290,6 +290,11 @@
 	<SUPR> =   206;
 	<HYPR> =   207;
 
+        <Yellow>  = 406;
+        <Blue>    = 407;
+        <Red>     = 408;
+        <Green>   = 409;
+
 	indicator 1  = "Caps Lock";
 	indicator 2  = "Num Lock";
 	indicator 3  = "Scroll Lock";
diff -aruN xkeyboard-config-2.11-orig/symbols/inet xkeyboard-config-2.11/symbols/inet
--- xkeyboard-config-2.11-orig/symbols/inet	2017-06-27 15:41:14.740375287 -0400
+++ xkeyboard-config-2.11/symbols/inet	2017-06-27 15:42:24.621504340 -0400
@@ -224,6 +224,11 @@
     key <FK16>   {      [ XF86Launch7       ]       };
     key <FK17>   {      [ XF86Launch8       ]       };
     key <FK18>   {      [ XF86Launch9       ]       };
+
+    key <Yellow> {      [ XF86Yellow        ]       };
+    key <Blue>   {      [ XF86Blue          ]       };
+    key <Red>    {      [ XF86Red           ]       };
+    key <Green>  {      [ XF86Green         ]       };
 };
 
 
